EIPS/eip-1.md

```diff
@@ -266,6 +286,79 @@ Many EIPs are written and maintained by developers with write access to the Ethe
 
 The editors don't pass judgment on EIPs. We merely do the administrative & editorial part.
 
+## Hardfork Meta
```

Comment:
I've read this section multiple times, and I think it could be more focused on explaining what the Hardfork Meta EIP is (as it does starting at [### Specification](https://github.com/ethereum/EIPs/pull/2508/files#diff-2f1874f8ac5135980a12a961cfafc079R297)) rather than a general overview of hardforks/upgrades. 

TL;DR, explaining that the way we track which EIP goes into upgrades is by the Hardfork Meta EIP, and perhaps linking a few. The discussion about upgrades vs. hardforks feels like it should be another section, and I'm not 100% sure it should be part of EIP-1. See my comment below :-)

Reply:
I wrote this section afterwards because I realized without some context it is difficult to understand the section is needed. This is an opportunity to have a source of truth for the community to reference. I don't think it needs to be in great detail, but some information is good to have to keep a 1one-stop place for understanding the EIP process and Ethereum Governance.

Otherwise, someone who is new would google right after, "What is a hardfork?" and learn who knows what from who knows where. Better to give them some grounding before they go off to learn deeper.